### ISSUE TITLE
Cannot read property 'algolia' of undefined

### ISSUE THREAD

**Issue by herrbischoff** [vuejs/vuepress#228]
## Environment

```
$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.13.4
BuildVersion:	17E199
```

```
$ node -v
v8.10.0
```

```
$ vuepress -V
0.8.0
```

## Issue

Just installed with `yarn global add vuepress` and followed the [Getting Started](https://vuepress.vuejs.org/guide/getting-started.html) instructions. When running `vuepress dev` or `vuepress build` the following error is thrown:

```
Extracting site metadata...TypeError: Cannot read property 'algolia' of undefined
    at resolveOptions (/Users/mbischoff/.config/yarn/global/node_modules/vuepress/lib/prepare.js:124:28)
    at prepare (/Users/mbischoff/.config/yarn/global/node_modules/vuepress/lib/prepare.js:24:25)
    at dev (/Users/mbischoff/.config/yarn/global/node_modules/vuepress/lib/dev.js:21:25)
    at args (/Users/mbischoff/.config/yarn/global/node_modules/vuepress/bin/vuepress.js:104:12)
    at Command.program.command.description.option.option.action (/Users/mbischoff/.config/yarn/global/node_modules/vuepress/bin/vuepress.js:31:21)
    at Command.listener (/Users/mbischoff/.config/yarn/global/node_modules/commander/index.js:315:8)
    at emitTwo (events.js:126:13)
    at Command.emit (events.js:214:7)
    at Command.parseArgs (/Users/mbischoff/.config/yarn/global/node_modules/commander/index.js:651:12)
    at Command.parse (/Users/mbischoff/.config/yarn/global/node_modules/commander/index.js:474:21)
```

**Comment by edm00se** [comment]
I got the same. Rolling back to version 0.7.1 instead of the freshly released 0.8.0 works for me.

**Comment by yyx990803** [comment]
Fixed in 0.8.1

**Comment by herrbischoff** [comment]
Wow, that was quick! :) I can confirm this is now working for me. Thanks!
